src/App.tsx

```diff
@@ -1,9 +1,10 @@
-import './App.css';
+import "./App.css";
+import MainPage from "./Pages/MainPage/MainPage";
```

Comment:
Shouldn't there be .tsx extension in the end?

Reply:
Hmm.. It auto imports without the extension and works fine